test/extended/prometheus/prometheus.go

```diff
@@ -49,6 +51,93 @@ type TelemeterClientConfig struct {
 	Enabled *bool `json:"enabled"`
 }
 
+// Set $MONITORING_AUTH_TEST_NAMESPACE to focus on the targets from a single namespace
+var monitoringAuthTestNamespace = os.Getenv("MONITORING_AUTH_TEST_NAMESPACE")
+
+var _ = g.Describe("[sig-instrumentation][Late] Platform Prometheus targets [apigroup:image.openshift.io]", func() {
```

Comment:
nit: not sure why `[apigroup:image.openshift.io]`

Reply:
Just followed the existing cases.
Removed in the recent patch.